### PR TITLE
doc/rados: edit pools.rst (4 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit doc/operations/rados/pools.rst.

https://tracker.ceph.com/issues/58485

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/pools.rst [hunk 1]
```diff
@@ -604,47 +604,38 @@ You may set values for the following keys:
...
 .. _recovery_priority:
 
 .. describe:: recovery_priority
-
-   When a value is set it will increase or decrease the computed
-   reservation priority. This value must be in the range -10 to
-   10.  Use a negative priority for less important pools so they
-   have lower priority than any new pools.
+   
+   :Description: Setting this value adjusts a pool's computed reservation priority.  reservation priority. This value must be in the range ``-10`` to ``10``. Any pool assigned a negative value will be given a lower priority than any new pools, so users are directed to assign negative values to low-priority pools.
```
Comment (colemitchell) [inline doc/rados/operations/pools.rst:638]: @zdover23 - Delete the extra _ reservation priority._

File: doc/rados/operations/pools.rst [hunk 2]
```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
...
 
 :Type: Integer
 
 
 Setting the Number of RADOS Object Replicas
 ===========================================
 
-To set the number of object replicas on a replicated pool, execute the following:
+To set the number of object replicas on a replicated pool, run the following
+command:
```
Comment (colemitchell) [inline doc/rados/operations/pools.rst:833]: @zdover23 - This should probably be _run a command of the following form_.

File: doc/rados/operations/pools.rst [hunk 3]
```diff
@@ -604,47 +604,38 @@ You may set values for the following keys:
...
-
 .. _scrub_min_interval:
 
 .. describe:: scrub_min_interval
-
-   The minimum interval in seconds for pool scrubbing when
-   load is low. If it is 0, the value osd_scrub_min_interval
-   from config is used.
+   
+   :Description: Sets the minimum interval (in seconds) for pool scrubbing when load is low. If the default value of ``0`` is in effect, then the value of ``osd_scrub_min_interval`` from central config is used.
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:611]: for successive scrubs of the pool's PGs when the load is low.

File: doc/rados/operations/pools.rst [hunk 4]
```diff
@@ -604,47 +604,38 @@ You may set values for the following keys:
...
 
 .. _scrub_max_interval:
 
 .. describe:: scrub_max_interval
-
-   The maximum interval in seconds for pool scrubbing
-   irrespective of cluster load. If it is 0, the value
-   osd_scrub_max_interval from config is used.
+   
+   :Description: Sets the maximum interval (in seconds) for pool scrubbing regardless of cluster load. If the value of ``scrub_max_interval`` is ``0``, then the value ``osd_scrub_max_interval`` from central config is used.
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:620]: for scrubs of the pool's PGs regardless

File: doc/rados/operations/pools.rst [hunk 5]
```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
...
 
    ceph osd pool set data size 3
 
-You may execute this command for each pool. **Note:** An object might accept
-I/Os in degraded mode with fewer than ``pool size`` replicas.  To set a minimum
-number of required replicas for I/O, you should use the ``min_size`` setting.
-For example:
+You may run the above command for each pool. 
+
+.. Note:: An object might accept I/Os in degraded mode with fewer than ``pool
```
Comment (colemitchell) [inline doc/rados/operations/pools.rst:850]: @zdover23 - Should _degraded_ be enclosed in double backticks?

File: doc/rados/operations/pools.rst [hunk 6]
```diff
@@ -604,47 +604,38 @@ You may set values for the following keys:
...
    :Default: ``0``
 
 .. _deep_scrub_interval:
 
 .. describe:: deep_scrub_interval
-
-   The interval in seconds for pool “deep” scrubbing. If it
-   is 0, the value osd_deep_scrub_interval from config is used.
+   
+   :Description: Sets the interval (in seconds) for pool “deep” scrubbing. If the value of ``deep_scrub_interval`` is ``0``, the value ``osd_deep_scrub_interval`` from central config is used.
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:629]: for deep scrubs of the pool's PGs

File: doc/rados/operations/pools.rst [hunk 7]
```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
 .. _recovery_op_priority:
 
 .. describe:: recovery_op_priority
-
-   Specify the recovery operation priority for this pool instead of :confval:`osd_recovery_op_priority`.
+   
+   :Description: Sets the recovery operation priority for a specific pool. This overrides the general priority determined by :confval:`osd_recovery_op_priority`.
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:648]: specific pool's PGs.

File: doc/rados/operations/pools.rst [hunk 8]
```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
...
+:Description: See recovery_op_priority_.
 
 :Type: Integer
 
 
 Setting the Number of RADOS Object Replicas
 ===========================================
 
-To set the number of object replicas on a replicated pool, execute the following:
+To set the number of object replicas on a replicated pool, run the following
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:832]: s/object/data/

File: doc/rados/operations/pools.rst [hunk 9]
```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
...
 .. prompt:: bash $
 
    ceph osd pool set {poolname} size {num-replicas}
 
-.. important:: The ``{num-replicas}`` includes the object itself.
-   If you want the object and two copies of the object for a total of
-   three instances of the object, specify ``3``.
-
-For example:
+.. important:: The ``{num-replicas}`` argument includes the object itself.  For
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:839]: s/object/primary object/

File: doc/rados/operations/pools.rst [hunk 10]
```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
...
+To get the number of object replicas, run the following command:
 
 .. prompt:: bash $
 
    ceph osd dump | grep 'replicated size'
 
-Ceph will list the pools, with the ``replicated size`` attribute highlighted.
-By default, ceph creates two replicas of an object (a total of three copies, or
-a size of 3).
+Ceph will list the pools and highlight the ``replicated size`` attribute.  By
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:872]: s/the//

**Review by anthonyeleven — APPROVED** [review]
A number of suggestions, non-blocking as per your judgement

File: doc/rados/operations/pools.rst (hunk 5 above)
Comment (anthonyeleven) [reply doc/rados/operations/pools.rst:850]: I don't think so in this caase

File: doc/rados/operations/pools.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:638]: Good catch. Corrected.

File: doc/rados/operations/pools.rst [hunk 11]
```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
...
    :Default: ``0``
 
 
-Get Pool Values
-===============
+Getting Pool Values
+===================
 
-To get a value from a pool, execute the following:
+To get a value from a pool's key, run the following command:
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rados/operations/pools.rst:657]: Roger that. Good catch.

File: doc/rados/operations/pools.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:833]: Good catch.

File: doc/rados/operations/pools.rst (hunk 3 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:611]: Accepted.

File: doc/rados/operations/pools.rst (hunk 4 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:620]: Accepted.

File: doc/rados/operations/pools.rst (hunk 6 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:629]: Accepted.

File: doc/rados/operations/pools.rst (hunk 7 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:648]: Accepted.

File: doc/rados/operations/pools.rst (hunk 8 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:832]: Accepted.

File: doc/rados/operations/pools.rst (hunk 9 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:839]: Accepted.

File: doc/rados/operations/pools.rst (hunk 10 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:872]: Accepted.

File: doc/rados/operations/pools.rst [hunk 12]
```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
...
+   should use the ``min_size`` setting.  For example, you might run the
+   following command:
 
 .. prompt:: bash $
 
    ceph osd pool set data min_size 2
 
-This ensures that no object in the data pool will receive I/O with fewer than
-``min_size`` replicas.
+This comamnd ensures that no object in the data pool will receive I/O if it has
```
Comment (zdover23) [inline doc/rados/operations/pools.rst:859]: s/comamnd/command/

Geez.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/51970 - Reef backport
https://github.com/ceph/ceph/pull/51971 - Quincy backport

**Pull request merged**